Makefile

```diff
@@ -34,6 +34,10 @@ doc:
 gh-pages:
 	bash .docgen.sh
 
+# requires ocamlformat
```

Comment:
Do we need to add this as a dependency to the opam file?

Reply:
I don't think there are any dev-dependencies in our opam files. We could add ocamlformat and merlin to all the opam files.